crates/sui-core/src/unit_tests/data/move_random/sources/move_random.move

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module examples::move_random {
+    // simple infinite loop to go out of gas in computation
+    public entry fun loopy() {
+        loop { }
```

Comment:
@tzakian @oxade any idea how to build a function that goes out of gas in a pretty short time with debug build?
Also do we know how long does it take for an infinite loop and max budget to terminate in a release build?

Reply:
I think emitting big events in a loop os probably the quickest way. Some maybe-useful code here https://github.com/MystenLabs/sui/blob/main/crates/sui-benchmark/src/workloads/data/adversarial/sources/adversarial.move#L90